Tasks/AzurePowerShellV3/AzurePowerShell.ps1

```diff
@@ -74,10 +78,23 @@ catch
 
 Update-PSModulePathForHostedAgent -targetAzurePs $targetAzurePs -authScheme $authScheme
 
+# troubleshoot link
+$troubleshoot = "https://aka.ms/azurepowershelltroubleshooting"
 try {
     # Initialize Azure.
     Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
     Initialize-Azure -azurePsVersion $targetAzurePs -strict
+    Write-Host "## Initializing Azure Complete"
+    $success = $true
+} 
+finally {
+    if (!$success) {
+        Write-VstsTaskError "Initialize Azure failed: For troubleshooting, refer: $troubleshoot"
+    }
```

Comment:
there should also be a message saying "Azure initialization complete"

Reply:
have added this log "Write-Host "## Initializing Azure Complete"" after Initialize-Azure is completed